camera_class.py

```
@@ -1,60 +1,341 @@
-                if i >= 60:
-                    logging.info("Loop ended, restarting camera resources…")
+        logging.debug("DONE PREFILL - entering main loop")
+
+        # --- Main capture loop ---
+        while not self.shutdown_flag.is_set():
+            logging.debug(f"Queue type: {type(self.q)}, queue id: {id(self.q)}, length: {len(self.q)}")
+            try:
+                # Handle pause event
+                if self.pause_event.is_set():
+                    if len(self.q):
+                        logging.info(f"Pausing queue for {pause_secs} seconds and clearing all frames [{len(self.q)}]")
```

Flake8 (7.2.0)

[error] undefined name 'pause_secs'

(F821)